DocNetwork DN-Dev-Availability-Service.postman_collection.json

```diff
@@ -112,10 +112,235 @@
+							"basic": [
+								{
+									"key": "username",
+									"value": "{{jira_email}}",
+									"type": "string"
+								},
+								{
+									"key": "password",
+									"value": "{{jira_api_key}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "GET",
+						"header": [
+							{
+								"key": "user",
+								"value": "",
+								"type": "text",
+								"disabled": true
+							},
+							{
+								"key": "Accept",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"url": {
+							"raw": "https://{{jira_org_domain}}/rest/api/3/issuetype/project?projectId=10008",
+							"protocol": "https",
+							"host": [
+								"{{jira_org_domain}}"
+							],
+							"path": [
+								"rest",
+								"api",
+								"3",
+								"issuetype",
+								"project"
+							],
+							"query": [
+								{
+									"key": "projectId",
+									"value": "10008"
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get All Projects",
+					"request": {
+						"auth": {
+							"type": "basic",
+							"basic": [
+								{
+									"key": "username",
+									"value": "{{jira_email}}",
+									"type": "string"
+								},
+								{
+									"key": "password",
+									"value": "{{jira_api_key}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "GET",
+						"header": [
+							{
+								"key": "user",
+								"value": "{{jira_email}}:{{jira_api_key}}",
+								"type": "text",
+								"disabled": true
+							},
+							{
+								"key": "Accept",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"url": {
+							"raw": "https://{{jira_org_domain}}/rest/api/3/project",
+							"protocol": "https",
+							"host": [
+								"{{jira_org_domain}}"
+							],
+							"path": [
+								"rest",
+								"api",
+								"3",
+								"project"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get Project Issues",
+					"request": {
+						"auth": {
+							"type": "basic",
+							"basic": [
+								{
+									"key": "username",
+									"value": "{{jira_email}}",
+									"type": "string"
+								},
+								{
+									"key": "password",
+									"value": "{{jira_api_key}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "https://{{jira_org_domain}}/rest/api/2/search?jql=project=AP&maxResults=1&issueId=AP-5025",
+							"protocol": "https",
+							"host": [
+								"{{jira_org_domain}}"
+							],
+							"path": [
+								"rest",
+								"api",
+								"2",
+								"search"
+							],
+							"query": [
+								{
+									"key": "jql",
+									"value": "project=AP"
+								},
+								{
+									"key": "maxResults",
+									"value": "1"
+								},
+								{
+									"key": "issueId",
+									"value": "AP-5025"
```

Comment:
Same with this ticket id

Reply:
Is this something we assume they'll change on using the Request?  Or should it also be made a variable?

Reply:
I agree this should be made a variable